src/web/security/core/pom.xml

```diff
@@ -59,6 +59,11 @@
 			<artifactId>mockrunner</artifactId>
 			<scope>test</scope>
 		</dependency>
+            <dependency>
+                <groupId>org.apache.wicket</groupId>
+                <artifactId>wicket</artifactId>
+                <type>pom</type>
+            </dependency>
```

Comment:
So the pom dependency is added everywhere because otherwise it does not get inherited? This seems quite inconvenient (and not obvious), wouldl't it be better to go back depending on actual jars instead? 
We'd have to upgrade the jar list when we upgrade Wicket, but at least, that would happen in a single place (as opposed to have to remember adding this dependency from each new module with a UI)